ambry-api/src/main/java/com.github.ambry/config/ClusterMapConfig.java

```diff
@@ -192,7 +192,7 @@ public ClusterMapConfig(VerifiableProperties verifiableProperties) {
         verifiableProperties.getIntInRange("clustermap.fixedtimeout.disk.retry.backoff.ms", 10 * 60 * 1000, 1,
             30 * 60 * 1000);
     clusterMapFixedTimeoutReplicaErrorThreshold =
-        verifiableProperties.getIntInRange("clustermap.fixedtimeout.replica.error.threshold", 1, 1, 100);
+        verifiableProperties.getIntInRange("clustermap.fixedtimeout.replica.error.threshold", 1, 1, Integer.MAX_VALUE);
```

Comment:
Change this to `Integer.MAX_VALUE` with intention to disable replica state policy if needed.